portfolio/pom.xml

```diff
@@ -23,8 +23,15 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.8.6</version>
+    </dependency>
   </dependencies>
```

Comment:
You are nor using gson. Is this for a follow up PR? ok to leave it, if you plan to use it.

Reply:
I did plan on using this for the next PR (forgot to switch branches when I implemented it), so I'll leave it be (Done)